### ISSUE TITLE
Segmentation fault in CircleCI

### ISSUE THREAD

**Issue by markus-stoll** [CognitionGuidedSurgery/msml#240]
Segementation fault in test:
test_boxROIToIndexOperators (msmltest.OperatorTest) ... share/run_test.sh: line 14: 32213 Segmentation fault  

https://circleci.com/gh/CognitionGuidedSurgery/msml/560

**Comment by markus-stoll** [comment]
strange - other branches arte almost identical. Trying again without cache...

**Comment by markus-stoll** [comment]
Trying again without cache broke it:

```
CMake Error at /usr/share/cmake-2.8/Modules/FindVTK.cmake:135 (message):
  VTK not found.  Set the VTK_DIR cmake cache entry to the directory
  containing VTKConfig.cmake.  This is either the root of the build tree, or
  PREFIX/lib/vtk for an installation.  For VTK 4.0, this is the location of
  UseVTK.cmake.  This is either the root of the build tree or
  PREFIX/include/vtk for an installation.
Call Stack (most recent call first):
  CMakeLists.txt:41 (FIND_PACKAGE)


-- Configuring incomplete, errors occurred!
See also "/home/ubuntu/msml/cbuild/CMakeFiles/CMakeOutput.log".
+ make -j 3
make: *** No targets specified and no makefile found.  Stop. 
```

**Comment by markus-stoll** [comment]
Does anyone know what the necsessary steps are to build with a cleared cache?

**Comment by markus-stoll** [comment]
Merging the branch to experimental causes the segmenttion fault also on experimental branch.
Merge was reverted - experimental branch is fixed.

**Comment by markus-stoll** [comment]
fixed 1391af1fda7c91bfcee9082b00166b89317b3765
having a function defined in .h file without an implementation causes a segmentation fault, if the .h file is wrapped by SWIG.
